src/components/Dashboard/FilingHistoryList/subtitles/FutureEffectiveFiledAndPendingCoa.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="future-effective-filed-and-pending-coa">
+    <span>FILED AND PENDING <FiledLabel :filing="filing" /></span>
+
+    <v-tooltip top content-class="pending-tooltip">
+      <template v-slot:activator="{ on }">
+        <div class="pending-alert" v-on="on">
+          <v-icon color="orange darken-2">mdi-alert</v-icon>
+        </div>
+      </template>
+      The updated office addresses will be legally effective on
+      {{ effectiveDateTime }}. No other filings are allowed until then.
+    </v-tooltip>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { ApiFilingIF } from '@/interfaces'
+import { DateUtilities } from '@/services'
+import FiledLabel from '../FiledLabel.vue'
+
+@Component({
+  components: { FiledLabel }
+})
+export default class FutureEffectiveFiledAndPendingCoa extends Vue {
+  @Prop({ required: true }) readonly filing!: ApiFilingIF
+  @Prop({ required: true }) readonly index!: number
+
+  /** The effective date-time of this filing. */
+  get effectiveDateTime (): string {
+    return this.filing.effectiveDate
+      ? DateUtilities.dateToPacificDateTime(new Date(this.filing.effectiveDate))
+      : '[unknown]'
+  }
```

Comment:
To reduce duplication, what do you think about creating a store getter to replace this method?

Reply:
We'd have to pass the filing itself, which is not so much a store getter as a utility function... in a mixin? I think doing that would be as ugly as the current code. How much do you want this, in this ticket?